deps/aro/aro/Driver.zig

```diff
@@ -710,7 +710,7 @@ fn processSource(
             d.comp.target.ofmt.fileExt(d.comp.target.cpu.arch),
         };
         break :blk d.output_name orelse
-            std.fmt.bufPrint(&name_buf, fmt_template, fmt_args) catch return d.fatal("Filename too long for filesystem: " ++ fmt_template, fmt_args);
+            (std.fmt.bufPrint(&name_buf, fmt_template, fmt_args) catch return d.fatal("Filename too long for filesystem: " ++ fmt_template, fmt_args));
```

Comment:
I think it's valuable and intuitive to chain `orelse` and `catch` together. Let's save this for a follow-up proposal, if any.

Reply:
in that case should one of them have their precedence raised?

Reply:
I'm suggesting that the line commented on here should be unchanged from how it is currently in master branch.